src/Index.tsx

```diff
@@ -76,8 +86,18 @@ const i18nPromise = initi18n();
   // Make sure localization is loaded
   await i18nPromise;
 
-  // Settings depends on i18n
-  watchLanguageChanges2(store);
+  // Update the language in both i18n and local storage when the user
+  // changes the language setting.
+  store.dispatch(
```

Comment:
Would still be nice for this to be in another file

Reply:
No worries, I will put it back in a settings file and use the same create pattern I used for the CSS ones